README.md

```diff
@@ -121,7 +121,17 @@ If you're using macOS, you can install Cog using Homebrew:
 brew install cog
 ```
 
-You can also download and install the latest release of Cog
+You can run an install script which will download and install the latest release of Cog:
+```
+# via curl
+sh -c "$(curl -fsSL https://raw.githubusercontent.com/replicate/cog/main/tools/install.sh)"
```

Comment:
Would `curl {URL} | sh` also work?

That might be a bit cleaner, but maybe there are some tradeoffs there?

Reply:
Could also use a short URL like `https://replicate.fyi/install-cog` with a redirect to this longer URL.

Reply:
```
curl -fsSL https://cog.run/installer | sh
```

Reply:
Yes, for some reason I thought the construction using a pipe wouldn't work with wget, but I just tested and it does.

(N.B. Part of the motivation for making this script in the first place was that my friend tried to use the existing instructions and one of his frustrations was that Ubuntu didn't have curl installed by default.)